fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/plugin/SimpleValueJqwikPlugin.java

```diff
@@ -218,12 +219,19 @@ public JavaStringConstraint generateStringConstraint(ArbitraryGeneratorContext c
 
 		@Override
 		public JavaIntegerConstraint generateIntegerConstraint(ArbitraryGeneratorContext context) {
-			return new JavaIntegerConstraint(
-				BigInteger.valueOf(this.positiveMinNumberValue),
-				BigInteger.valueOf(this.positiveMaxNumberValue),
-				BigInteger.valueOf(this.negativeMinNumberValue),
-				BigInteger.valueOf(this.negativeMaxNumberValue)
-			);
+			BigInteger positiveMin = BigInteger.valueOf(this.positiveMinNumberValue);
+			BigInteger positiveMax = BigInteger.valueOf(this.positiveMaxNumberValue);
+			BigInteger negativeMax = BigInteger.valueOf(this.negativeMinNumberValue);
+			BigInteger negativeMin = BigInteger.valueOf(this.negativeMaxNumberValue);
+
+			Type type = context.getResolvedType();
+
+			if ((type == Byte.class || type == byte.class)) {
+				positiveMax = BIG_INTEGER_MAX_BYTE;
+				negativeMax = BIG_INTEGER_MIN_BYTE;
```

Comment:
It should be required to be compatible with the option.

`negativeMax = Math.max(negativeMax, BIG_INTEGER_MIN_BYTE)`